<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstRun</class>
 <widget class="QWidget" name="FirstRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>600</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageWelcome">
    <property name="styleSheet">
     <string notr="true">Image:  url(:/images/FirstRunWelcome.png);</string>
    </property>
    <widget class="QLabel" name="welcomeTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>275</y>
       <width>1024</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>Welcome to Estonian eID application!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="welcomeDescription">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>315</y>
       <width>1024</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>DigiDoc is designed for managing eIDs in Estonia and is the official application for signing and encryption of documents</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="welcomeSelectLang">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>1024</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>20</height>
      </size>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>Vali suhtluskeel / Выберите язык / Select language</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="welcomeLanguage">
     <property name="geometry">
      <rect>
       <x>411</x>
       <y>415</y>
       <width>201</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>201</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>201</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	image: none;
    border: 1px solid #DEE4E9;
    border-radius: 2px;
    background-color: #FFFFFF;
	color: #353739;
}

/* list */
QComboBox QAbstractItemView {
	image: none;
    border: 1px solid #DEE4E9;
    border-radius: 2px;
	padding: 3px 3px 3px 3px;
    background-color: #FFFFFF;
}

/* rows */
QComboBox:item {
    background-color: #FFFFFF;
	color: #353739;
}

/* selected row */
QComboBox:item:selected {
    background-color: #006EB5;
    color: #FFFFFF;
}

/* drop-down button*/
QComboBox::drop-down {
    width: 0px;
	border: none;
}

/* drop-down button image*/
QComboBox::down-arrow {
	border: none;
    image: none;
}</string>
     </property>
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
     <property name="maxCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QPushButton" name="welcomeContinue">
     <property name="geometry">
      <rect>
       <x>412</x>
       <y>458</y>
       <width>200</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>34</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	image: none;
	border-radius: 2px;
	border: none;
	color: #ffffff;
	background-color: #006EB5;
}
QPushButton:pressed {
	background-color: #41B6E6;
}
QPushButton:hover:!pressed {
	background-color: #008DCF;
}
QPushButton:disabled {
	background-color: #BEDBED;
}</string>
     </property>
     <property name="text">
      <string>CONTINUE</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageIntro">
    <property name="styleSheet">
     <string notr="true">Image:  url(:/images/FirstRunIntro.png);</string>
    </property>
    <widget class="QPushButton" name="viewSigning">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>418</y>
       <width>298</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>34</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	image: none;
	border-radius: 2px;
	border: none;
	color: #ffffff;
	background-color: #006EB5;
}
QPushButton:pressed {
	background-color: #41B6E6;
}
QPushButton:hover:!pressed {
	background-color: #008DCF;
}
QPushButton:disabled {
	background-color: #BEDBED;
}</string>
     </property>
     <property name="text">
      <string>VIEW INTRODUCTION</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="viewEncryption">
     <property name="geometry">
      <rect>
       <x>363</x>
       <y>418</y>
       <width>298</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>34</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	image: none;
	border-radius: 2px;
	border: none;
	color: #ffffff;
	background-color: #006EB5;
}
QPushButton:pressed {
	background-color: #41B6E6;
}
QPushButton:hover:!pressed {
	background-color: #008DCF;
}
QPushButton:disabled {
	background-color: #BEDBED;
}</string>
     </property>
     <property name="text">
      <string>VIEW INTRODUCTION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewEid">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>418</y>
       <width>298</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>34</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	image: none;
	border-radius: 2px;
	border: none;
	color: #ffffff;
	background-color: #006EB5;
}
QPushButton:pressed {
	background-color: #41B6E6;
}
QPushButton:hover:!pressed {
	background-color: #008DCF;
}
QPushButton:disabled {
	background-color: #BEDBED;
}</string>
     </property>
     <property name="text">
      <string>VIEW INTRODUCTION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="introSkip">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>525</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>25</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	image: none;
	border-radius: 2px;
	border: none;
	color: #006EB5;
}
QPushButton:pressed {
	color: #FFFFFF;
	background-color: #006EB5;
}
QPushButton:hover:!pressed {
	border: 1px solid #006EB5;
	color: #006EB5;
}
QPushButton:disabled {
	color: #75787B;
	background-color: #BEDBED;
}</string>
     </property>
     <property name="text">
      <string>Skip introductions</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="introTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>1024</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>The DigiDoc application allows you:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="introSigningTitle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>297</y>
       <width>298</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>Sign documents</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="introEncryptingTitle">
     <property name="geometry">
      <rect>
       <x>363</x>
       <y>297</y>
       <width>298</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>Encrypt documents</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="introMyIDsTitle">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>297</y>
       <width>298</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>Manage your IDs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="introSigningBrief">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>298</width>
       <height>65</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>A brief summary and introductory text for signing a document</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="introEncryptingBrief">
     <property name="geometry">
      <rect>
       <x>363</x>
       <y>340</y>
       <width>298</width>
       <height>65</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>A brief summary and introductory text for encrypting documents</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="introMyIDsBrief">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>340</y>
       <width>298</width>
       <height>65</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;</string>
     </property>
     <property name="text">
      <string>A brief summary and introductory text for Managing your IDs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageSigning">
    <property name="styleSheet">
     <string notr="true">Image:  url(:/images/FirstRunSigning.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="pageEncrypt">
    <property name="styleSheet">
     <string notr="true">Image:  url(:/images/FirstRunEncrypt.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="pageMyEID">
    <property name="styleSheet">
     <string notr="true">Image:  url(:/images/FirstRunMyEID.png);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
